src/components/SimpleFeedback.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { graphql, useStaticQuery } from 'gatsby';
+import { css } from '@emotion/core';
+import { Button, Icon, PageTools } from '@newrelic/gatsby-theme-newrelic';
+
+const SimpleFeedback = ({ pageTitle }) => {
+  const { site } = useStaticQuery(graphql`
+    query FeedbackQuery {
+      site {
+        siteMetadata {
+          repository
+        }
+      }
+    }
+  `);
+
+  const issueUrl = `${site.siteMetadata.repository}/issues/new`;
+  const labels = `content,feedback`;
+  const title = pageTitle
+    ? `Feedback+for:+${pageTitle.replace(' ', '+')}`
+    : 'Website Feedback';
+
+  const positiveFeedback = `${issueUrl}?labels=${labels},feedback-positive&title=${title}`;
+  const negativeFeedback = `${issueUrl}?labels=${labels},feedback-negative&title=${title}`;
+
+  return (
+    <PageTools.Section>
+      <PageTools.Title>Feedback</PageTools.Title>
+      <div
+        css={css`
+          font-size: 0.875rem;
+        `}
+      >
+        Was this page helpful?
+      </div>
+      <div
+        css={css`
+          display: flex;
+          margin-top: 0.5rem;
+          justify-content: center;
```

Comment:
We should left align and stretch these buttons out (similar to how we do with the other GitHub buttons above this).